src/internal/connector/onedrive/collection_test.go

```diff
@@ -267,8 +283,10 @@ func (suite *CollectionUnitTestSuite) TestCollection() {
 			test.expectErr(t, err)
 
 			if err != nil {
-				if len(test.expectLabel) > 0 {
-					assert.True(t, clues.HasLabel(err, test.expectLabel), "has clues label:", test.expectLabel)
+				if len(test.expectLabels) > 0 {
+					for _, label := range test.expectLabels {
+						assert.True(t, clues.HasLabel(err, label), "has clues label:", label)
+					}
```

Comment:
Conditional not needed.  nil and zero valued slices will get skipped naturally
```suggestion
					for _, label := range test.expectLabels {
						assert.True(t, clues.HasLabel(err, label), "has clues label:", label)
					}
```